idaes/models/properties/general_helmholtz/components/registry.py

```diff
@@ -15,12 +15,6 @@
 # TODO: Missing docstrings
 # pylint: disable=missing-function-docstring
```

Comment:
Seeing as you are editing these files, you should fix the doc string issue so we can remove this whilst we are at it.